erpnext/stock/doctype/item/item.js

```diff
@@ -812,6 +812,29 @@ frappe.ui.form.on("Item Supplier", {
 	}
 })
 
+frappe.ui.form.on("Shipping Information", {
+	box_capacity_for_item: function(frm, cdt, cdn) {
+		var row = locals[cdt][cdn];
+		if (row.box_capacity_for_item > row.max_box_capacity) {
+			frappe.throw(__("Box Capacity for Item cannot be greater than Maximum Box Capacity"))
+		}
+	}
+})
+
+cur_frm.fields_dict['shipping_information'].grid.get_field('uom').get_query = function(doc,cdt,cdn) {
```

Comment:
```suggestion
frm.set_query('shipping_information', 'uom', function(doc,cdt,cdn) {
```

Move this to refesh trigger